includes/forms/template.php

```diff
@@ -127,52 +114,34 @@ function give_get_donation_form( $args = array() ) {
 			 * @param int $form_id The form ID.
 			 * @param array $args An array of form arguments.
 			 */
-			do_action( 'give_pre_form', $form->ID, $args );
+			do_action('give_pre_form', $form->ID, $args, $form );
+
+			// Set form html tags.
+			$form_html_tags = array(
+				'id' => "give-form-{$form_id}",
+				'class' => $form_classes,
+				'action' => esc_url_raw( $form_action ),
+			);
+
+			/**
+			 * Filter the form html tags.
+			 *
+			 * @since 1.8.17
+			 */
+			$form_html_tags = apply_filters('give_form_html_tags', (array) $form_html_tags, $form );
```

Comment:
It can be like this 
$form_html_tags = (array) apply_filters('give_form_html_tags', $form_html_tags, $form );